pom.xml

```diff
@@ -788,6 +808,21 @@
     </pluginManagement>
 
     <plugins>
+      <plugin>
+        <groupId>net.revelc.code.formatter</groupId>
+        <artifactId>formatter-maven-plugin</artifactId>
+        <configuration>
+          <configFile>../jboss-java-formatter.xml</configFile>
```

Comment:
It is possible to reference the file remotely. Also, I think we must force the linefeed or it breaks on Windows and Mac.

```
<configuration>
            <configFile>https://raw.githubusercontent.com/project-ncl/ide-config/master/eclipse/jboss-java-formatter.xml</configFile>
            <lineEnding>LF</lineEnding>
          </configuration>
```

Reply:
OK, I'll use the remote file. What should I do with the linefeed?